src/unix/linux_like/linux/mod.rs

```diff
@@ -4708,6 +4708,43 @@ pub const RTNLGRP_MCTP_IFADDR: c_uint = 0x22;
 pub const RTNLGRP_TUNNEL: c_uint = 0x23;
 pub const RTNLGRP_STATS: c_uint = 0x24;
 
+// linux/cn_proc.h
+pub const PROC_CN_MCAST_LISTEN: c_int = 1;
+pub const PROC_CN_MCAST_IGNORE: c_int = 2;
+
+pub const PROC_EVENT_NONE: c_int = 0x00000000;
```

Comment:
There is a `c_enum!` macro for representing enums as a type plus constants, could you use that here?